presidio-anonymizer/Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 flask = "==1.1.2"
+pycryptodome = "==3.10.1"
```

Comment:
@omri374  Verifying that the licensing is valid for us. Its BSD-2, which is less restrictive than flask's BSD-3 for instance